static/templates/partials/iframely-widget-card.tpl

```diff
@@ -10,7 +10,7 @@
 
             <!-- IF title -->
             <h4 class="media-heading">
-                <a href="{embed.url}" target="_blank" rel="nofollow" class="one-line">
+                <a href="{embed.meta.canonical}" target="_blank" rel="nofollow" class="one-line">
```

Comment:
You are right. My fault. Iframely self-hosted doesn't provide it because its already in request path. However it can be added later...

So currently `meta.canonical` must be used.

Optionally original `url` can be passed from upper call like `data.match` did.